Cargo.toml

```diff
@@ -124,9 +124,9 @@ ledger-transport-hid = "0.10.0"
 libc = "0.2.97"
 libloading = "0.7.2"
 linkme = "0.3.24"
-# branch = "main"
-masp_primitives = { git = "https://github.com/anoma/masp", rev = "3aacc707c5948e7423589ac617305448bead9842" }
-masp_proofs = { git = "https://github.com/anoma/masp", rev = "3aacc707c5948e7423589ac617305448bead9842", default-features = false, features = ["local-prover"] }
+# branch = "murisi/receive-randomness-with-borsh2"
+masp_primitives = { git = "https://github.com/anoma/masp", rev = "3c9a61895ebf21ed9f60ce40c481ee932c719b71" }
+masp_proofs = { git = "https://github.com/anoma/masp", rev = "3c9a61895ebf21ed9f60ce40c481ee932c719b71", default-features = false, features = ["local-prover"] }
```

Comment:
Is this merged to main?

Reply:
This has now been merged to `main` and the commit hash has been updated accordingly.